src/test/java/org/kohsuke/github/GHPullRequestTest.java

```diff
@@ -421,6 +421,66 @@ public void setLabels() throws Exception {
         assertEquals(label, labels.iterator().next().getName());
     }
 
+    @Test
+    // Requires push access to the test repo to pass
+    public void addLabels() throws Exception {
+        GHPullRequest p = getRepository().createPullRequest("addLabels", "test/stable", "master", "## test");
+        String addedLabel1 = "addLabels_label_name_1";
+        String addedLabel2 = "addLabels_label_name_2";
+        String addedLabel3 = "addLabels_label_name_3";
+
+        p.addLabels(addedLabel1);
+        p.addLabels(addedLabel2, addedLabel3);
+
+        Collection<GHLabel> labels = getRepository().getPullRequest(p.getNumber()).getLabels();
+        assertEquals(3, labels.size());
+        assertThat(labels,
+                containsInAnyOrder(hasProperty("name", equalTo(addedLabel1)),
+                        hasProperty("name", equalTo(addedLabel2)),
+                        hasProperty("name", equalTo(addedLabel3))));
+    }
```

Comment:
```suggestion
        // Adding a label which is already present does not throw an error
        p.addLabels(addedLabel1);
    }
```

Reply:
Need to rerecord for this change, sorry.

Reply:
Frankly, I fail to see how this is related to this PR at all.

Reply:
Or is it because the way it was broken before, this case would work?